### PR TITLE
[Cleanup][Windowing] Improved resolution doxy info

### PR DESCRIPTION
**Pull request by CastagnaIT**
## Description

Add doxy info about resolution 

## Motivation and context

some RESOLUTION_INFO properties like iWidth/iScreenWidth can be misunderstood

## How has this been tested?

tested by generating doxygen

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
I've canceled the build, since this sort of depends on https://github.com/xbmc/xbmc/pull/22394 (just merged in o*). In those cases you can simply set the "No jenkins" label to avoid wasting build resources

File: xbmc/windowing/Resolution.h [hunk 1]
```diff
@@ -62,23 +62,25 @@ struct EdgeInsets
...
-  int iScreenWidth;
-  int iScreenHeight;
-  int iSubtitles;
-  uint32_t dwFlags;
-  float fPixelRatio;
-  float fRefreshRate;
-  std::string strMode;
-  std::string strOutput;
-  std::string strId;
+  bool bFullScreen; //!< Specify if it is a fullscreen resolution, otherwise windowed
```
Comment (lrusak) [inline xbmc/windowing/Resolution.h:71]: Can you put the comment above each member and add whitespace between each member?

Comment (CastagnaIT) [reply xbmc/windowing/Resolution.h]: it become a bit long but done

**Comment by fuzzard** [comment]
Jenkins build this please

**Review by garbear — APPROVED** [review]
Doxy improvements are always welcome, thanks! I don't see any problems with the new text.

**Pull request merged**